test/integrations/destinations/google_adwords_offline_conversions/dataDelivery/business.ts

```diff
@@ -0,0 +1,728 @@
+                    errors: [
+                      {
+                        errorCode: {
+                          offlineUserDataJobError: 'INVALID_SHA256_FORMAT',
+                        },
+                        message: 'The SHA256 encoded value is malformed.',
+                        location: {
+                          fieldPathElements: [
+                            {
+                              fieldName: 'operations',
+                              index: 0,
+                            },
+                            {
+                              fieldName: 'create',
+                            },
+                            {
+                              fieldName: 'user_identifiers',
+                              index: 0,
+                            },
+                            {
+                              fieldName: 'hashed_email',
+                            },
+                          ],
+                        },
+                      },
+                    ],
+                    requestId: '68697987',
+                  },
+                ],
+                message: 'Request contains an invalid argument.',
+                status: 'INVALID_ARGUMENT',
+              },
+            },
+            statTags: {
+              destType: 'GOOGLE_ADWORDS_OFFLINE_CONVERSIONS',
+              destinationId: 'default-destinationId',
+              errorCategory: 'network',
+              errorType: 'aborted',
+              feature: 'dataDelivery',
+              implementation: 'native',
+              module: 'destination',
+              workspaceId: 'default-workspaceId',
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'gaoc_v0_scenario_2',
+    name: 'google_adwords_offline_conversions',
+    description:
+      '[Proxy v0 API] :: Test for a valid request with a successful 200 response from the destination',
+    successCriteria: 'Should return 200 with no error with destination response',
+    scenario: 'Business',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: generateProxyV0Payload({
+          headers: headers.header1,
+          params: params.param1,
+          JSON: validRequestPayload1,
+          endpoint: 'https://googleads.googleapis.com/v14/customers/1112223333/offlineUserDataJobs',
+        }),
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: {
+          output: {
+            status: 200,
+            message:
+              '[Google Ads Offline Conversions Response Handler] - Request processed successfully',
+            destinationResponse: {
+              response: {
+                name: 'customers/111-222-3333/operations/abcd=',
+              },
+              status: 200,
+            },
+          },
+        },
+      },
+    },
+  },
+  {
+    id: 'gaoc_v0_scenario_3',
```

Comment:
Description and successCriteria is same for `gaoc_v0_scenario_2` and `gaoc_v0_scenario_3` can we update with more details about exact test case

Reply:
addressed